knudev-liquibase/src/main/resources/db/changelog/team_management/2024_10_11_init_db.yaml

```diff
@@ -31,4 +31,163 @@ databaseChangeLog:
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: create_specialty_table
+      author: DenysLeonenko
+      changes:
+        - createTable:
+            schemaName: team_management
+            tableName: specialty
+            columns:
+              - column:
+                  name: code_name
+                  type: double
+                  constraints:
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
 
+  - changeSet:
+      id: add_department_id_and_specialty_code_number_columns
+      author: DenysLeonenko
+      changes:
+        - addColumn:
+            schemaName: team_management
+            tableName: account_profile
+            columns:
+              - column:
+                  name: department_id
+                  type: uuid
+                  constraints:
+                    nullable: false
+              - column:
+                  name: specialtyCodeName
+                  type: double
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: create_account_department_table
+      author: DenysLeonenko
+      changes:
+        - createTable:
+            schemaName: team_management
+            tableName: account_department
+            columns:
+              - column:
+                  name: account_id
+                  type: integer
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: department_id
+                  type: uuid
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: create_account_specialty_table
+      author: DenysLeonenko
+      changes:
+        - createTable:
+            schemaName: team_management
+            tableName: account_specialty
+            columns:
+              - column:
+                  name: account_id
+                  type: integer
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: specialty_id
+                  type: double
+                  constraints:
+                    nullable: false
+
+  - changeSet:
+      id: create_departments_to_speciaties_table
+      author: DenysLeonenko
+      changes:
+        - createTable:
+            schemaName: team_management
+            tableName: departments_specialties
+            columns:
+              - column:
+                   name: specialties
+                   type: double
+                   constraints:
+                     nullable: false
+              - column:
+                  name: departments
+                  type: uuid
+                  constraints:
+                    nullable: false
```

Comment:
here department_id is PK, speicality_id is FK


So all of this id-id tables ( one-to-one, many-to-many and so on) you create ONE change set, where you
1. Create the table
2. Create columns in it
3. What now you dont do, is set constraints. One column of that table should be PK, and second should be FK, referencing some table. 

If making this without constrains, then tables wont be connected to each other, which will make no sense

Reply:
done